smeargle.py

```diff
@@ -177,103 +184,109 @@ def sd_api_call():
         image.save(f'test_image_output/{api_call_count}.png', pnginfo=metadata)
         print(f"image successfully created, tagged, and moved to test_image_output/{api_call_count}.png. sleeping for {sleep_timer}s")
     else:
-        image.save(f'{dirpath}\cover.png', pnginfo=metadata)
-        print(f"image successfully created, tagged, and moved to {dirpath}\cover.png. sleeping for {sleep_timer}s\n")
+        image.save(f'{dirpath}\\cover.png', pnginfo=metadata)
+        print(f"image successfully created, tagged, and moved to {dirpath}\\cover.png. sleeping for {sleep_timer}s\n")

```

Comment:
Pretty sure this needs an extra slash, was it working before?

Reply:
Yes it was working before. It threw errors about improper escapes but the script did run despite the errors. This was for Windows and I just copy pasted the path from File Explorer, hence the single slash. Good fix, will include.